services/web-app/components/external-links/external-links.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
```

Comment:
```suggestion
 * Copyright IBM Corp. 2022, 2022
```

Small detail, when creating a new file, both copyright years should be the current deal.